front/app/containers/Admin/dashboard/components/ChartFilters/ProjectFilter.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
```

Comment:
This used to be in `front/app/containers/Admin/dashboard/components/ChartFilters.tsx`, and the generateProjectOptions` logic used to be in `front/app/containers/Admin/dashboard/summary/index.tsx` and `front/app/containers/Admin/dashboard/users/index.tsx`